data/settings_list.yaml

```diff
@@ -616,7 +616,7 @@
     - "On"
   options:
     - "off": "Traps that spawn Groose will not be added to the item pool."
-    - "on": "Traps that spawn Groose can be picked when replacing items with traps (controlled by the Trap Mode setting). To prevent performance issues, Groose will not spawn in Silent Realms."
+    - "on": "Traps that spawn Groose can be picked when replacing items with traps (controlled by the Trap Mode setting). To prevent performance issues, Groose Traps will not spawn in Silent Realms."
```

Comment:
I think you forgot the second `Groose` in the suggestion xD. Now it sounds like Groose Traps _really_ won't be in silent realms. The suggestion was: `Groose Traps will not spawn Groose in Silent Realms.`

Reply:
Should be good now xD